### PR TITLE
Improve the Flex overflow marker style

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
Fixes https://github.com/flutter/flutter/issues/2387

**Comment by Hixie** [comment]
I need to also update the docs for Row to not say "red" anymore.

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -2613,12 +2613,13 @@ class Flex extends MultiChildRenderObjectWidget {
 ///
 /// ## Troubleshooting
 ///
-/// ### Why is my row turning red?
+/// ### Why does my row have a yellow and black warning stripe?
 ///
 /// If the contents of the row overflow, meaning that together they are wider
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/basic.dart]: nit: this sentence feels super awkward to read. It's so long and so many "and"s...

**Review by goderbauer — APPROVED** [review]

**Comment by Hixie** [comment]
I tweaked the text and added more details to Column while I was at it.

Will land on green unless you want to review it further and have any additional comments.

File: packages/flutter/lib/src/widgets/basic.dart [hunk 2]
```diff
@@ -2897,12 +2897,15 @@ class Flex extends MultiChildRenderObjectWidget {
...
 ///
-/// If the contents of the row overflow, meaning that together they are wider
-/// than the row, then the row runs out of space to give its [Expanded] and
-/// [Flexible] children, and reports this by drawing a red warning box on the
-/// edge that is overflowing.
+/// If the non-flexible contents of the row (those that are not wrapped in
+/// [Expanded] or [Flexible] widgets) are together wider than the row itself,
+/// then the row is said to have overflowed. When a row overflows, the row does
+/// not have any remaining space to share between its [Expanded] and [Flexible]
+/// children. The row reports this this by drawing a yellow and black striped
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/basic.dart]: remove one this?

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**